pages/checkout/index.html

```diff
@@ -1,15 +1,15 @@
 ---
 layout: default
-title: Become a Gruntwork Subscriber
-excerpt: Configure the plan just right for your team.
+title: Customize your plan with add-ons
+excerpt: We offer professional support and a reference architecture.
```

Comment:
Is `/checkout` the first page of the flow or the second now? This seems like the title for the second page, no?

Reply:
`/checkout` page is the second page of the flow now. `/pricing` is the first page. The title of the checkout page was changed in the new design. https://projects.invisionapp.com/share/EDSPCQC9QBX#/screens/370853456

Reply:
Roger